libminifi/include/capi/Plan.h

```diff
@@ -46,11 +48,50 @@
 #include "capi/cstructs.h"
 #include "capi/api.h"
 
+enum FailureStrategy { AS_IS, ROLLBACK };
+
+bool intToFailureStragey(int in, FailureStrategy *out);
+
+using callback_type = std::function<void(flow_file_record*)>;
```

Comment:
^I'm not inherently against aliasing but I think 'callback_type' does little to improve readability, and since readability is in the eye of the beholder ( and I don't have a preference if others use it ) is there a way to at least better indicate that this is a function used for a certain type of callback?

Reply:
Sure, I will rename this, should be failure callback type or something alike.